.github/workflows/pdk-basic.yml

```diff
@@ -0,0 +1,41 @@
+name: PDK basic
+
+on:
+  workflow_call:
+    inputs:
+      container_image:
+        description: Image to use when running tests
+        default: 'puppet/puppet-dev-tools:2022-04-21-e44b72b'
+        required: false
+        type: string
+
+jobs:
+  validate:
+    runs-on: ubuntu-latest
+    container: ${{ inputs.container_image }}
+    outputs:
+        puppet_unit_test_matrix: ${{ steps.get-outputs.outputs.puppet_unit_test_matrix }}
+        github_action_test_matrix: ${{ steps.get-outputs.outputs.github_action_test_matrix }}
+    steps:
+      - uses: actions/checkout@v2
+      - name: action-pdk-validate-puppet-7
+        run: pdk validate --puppet-version=7
```

Comment:
I'd make version 7 an input with a default as well. That allows users to trivially override it. Also means switching 7 to 8 is just a one line change instead of two.